packages/orun/orun.0.1.2/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "2.0"
+synopsis: "Run benchmarks and measure performance"
+maintainer: "Stephen Dolan <stephen.dolan@cl.cam.ac.uk>"
+authors: "Stephen Dolan <stephen.dolan@cl.cam.ac.uk>"
+license: "MIT"
+dev-repo: "git+https://github.com/ocaml-bench/orun.git"
+depends: [ "ocaml" "cmdliner" "yojson" ]
+build: [
+  ["ocaml-update-c" "wait4.c"] {ocaml:update-c}
+  ["dune" "build" "-p" name]
```

Comment:
```suggestion
  ["dune" "build" "-p" name "-j" jobs]
```
This allows users to select how many processes they want to use to build this package.